CONTRIBUTING.md

```diff
@@ -0,0 +1,282 @@
+# How to contribute
+
+## How to prepare
+
+* You need a [GitHub account](https://github.com/signup/free)
+* Submit an [issue ticket](https://github.com/tomato42/tlslite-ng/issues) for
+  your issue if there is none yet
+  * Describe the issue and include steps to reproduce if it's a bug, mention
+    the earliest version that you know is affected and the version you're using.
+  * Describe the enhancement and your general ideas on how to implement it
+    if you want to add new feature or extend existing one. This is not
+    necessary if the change is small.
+* If you are able and want to fix the issue, fork the repository on GitHub and
+  see [Developing changes](#developing-changes) section
+
+## Ways to contribute
+
+* Look for issues in the
+  [projects](https://github.com/tomato42/tlsfuzzer/projects) page.
+  [TLS 1.3 coverage](https://github.com/tomato42/tlsfuzzer/projects/1) is the
+  highest priority now.
+* Look for
+  [open issues](https://github.com/tomato42/tlsfuzzer/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22),
+  ones that have
+  [maintenance](https://github.com/tomato42/tlsfuzzer/issues?q=is%3Aissue+is%3Aopen+label%3Amaintenance)
+  label and/or
+  [good first issue](https://github.com/tomato42/tlsfuzzer/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
+  are good choices
+* Report ideas for
+  [new test scripts](https://github.com/tomato42/tlsfuzzer/issues/new?template=New_test_script.md)
+  to verify TLS implementaions with
+* Run static code analyses on tlsfuzzer, report issues they found
+  * Integrate them into CI or development scripts
+  * [yala](https://github.com/cemsbr/yala) may be a good place to start
+* Extend documentation (including this file!) or wiki
+  * document how to set up and run tlsfuzzer on Windows or OSX
+
+## Developing changes
+
+### TL;DR
+
+1. [Fork it](https://github.com/tomato42/tlsfuzzer/fork)
+1. Clone it
+   * `git clone git@github.com:<your-github-nick>/tlsfuzzer.git`
+   * `cd tlsfuzzer`
+1. Install dependencies (as root)
+   * `pip2 install coverage ecdsa epydoc unittest2 mock`
+   * `pip3 install coverage ecdsa`
+   * `pip2 install -r build-requirements.txt`
+   * `pip3 install -r build-requirements.txt`
+1. Download tlslite-ng
+   * `git clone https://github.com/tomato42/tlslite-ng.git .tlslite-ng`
+   * `ln -s .tlslite-ng/tlslite tlslite`
+1. Verify installation
+   * `make test`
+1. Create your feature branch (`git checkout -b my-new-feature`)
+1. Commit your changes (`git commit -am 'Add some feature'`)
+1. Run tests (`make test`)
```

Comment:
Maybe write what run tests will actually do; e.g., "Run tests (make test) to verify that new functionality works against tlslite.

Reply:
it is the TL;DR section so I'd rather omit this